packages/dd-trace/test/ci-visibility/dynamic-instrumentation/dynamic-instrumentation.spec.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+require('../../../../dd-trace/test/setup/tap')
+
+const { fork } = require('child_process')
+const path = require('path')
+
+const { assert } = require('chai')
+
+describe('test visibility with dynamic instrumentation', () => {
+  let childProcess
+
+  afterEach(() => {
+    if (childProcess) {
+      childProcess.kill()
+    }
+  })
+
+  it('can grab local variables', (done) => {
+    // Dynamic Instrumentation - Test Visibility not currently supported for windows
+    if (process.platform === 'win32') {
+      done()
+      return
+    }
+    childProcess = fork(path.join(__dirname, 'target-app', 'test-visibility-dynamic-instrumentation-script.js'))
+
+    childProcess.on('message', ({ snapshot: { language, stack, probe, captures }, snapshotId }) => {
+      assert.exists(snapshotId)
+      assert.exists(probe)
+      assert.exists(stack)
+      assert.equal(language, 'javascript')
+
+      assert.deepEqual(captures, {
+        lines: {
+          9: {
```

Comment:
Consider if you can re-use the [`getBreakpointInfo`](https://github.com/DataDog/dd-trace-js/blob/57f8a10ae8e521133ac929dc81a1c11e9ac52793/integration-tests/debugger/utils.js#L107-L124) function for you tests. That way, if you ever change the line your breakpoint is on, your tests just adapt automatically.

Reply:
the function needs some adjusting, as it's currently tailored for debugger tests (for example by the usage of `debugger/target-app/` and `__dirname`. The hardcoding of lines is no big issues right now, but I'll adapt the function once it becomes an issue 👍